modules/ROOT/nav.adoc

```diff
@@ -32,11 +34,11 @@
 * xref:manage-index.adoc[Manage Runtime Fabric]
  ** xref:view-health.adoc[View Runtime Fabric Health Details]
  ** xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric on VMs / Bare Metal]
- ** xref:using-opscenter.adoc[Monitor Anypoint Runtime Fabric]
  ** xref:install-rtfctl.adoc[Install the Runtime Fabric Command Line Tool]
  ** xref:install-patches.adoc[Perform Scheduled Maintenance on Runtime Fabric on VMs / Bare Metal]
  ** xref:replace-license-key.adoc[Replace a Mule License Key]
- ** xref:manage-proxy.adoc[Manage Proxies used by Runtime Fabric]
+ ** xref:manage-proxy-self.adoc[Manage Proxies on Runtime Fabric on Self-Managed Kubernetes]
+ ** xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric on VMs / Bare Metal]
```

Comment:
how about keep it consistent to either `Manage Proxies on`or `Manage Proxies Used by` from line above?

Reply:
manage proxies on